src/pages/proposal/completed/GeneralVote.tsx

```
@@ -38,6 +41,19 @@ const GeneralVote = ({ data, voteTypeName }: GeneralVoteProps) => {
           flex: '1 1 60%',
         }}
       >
+        <Typography
+          variant="body2"
+          dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)